underactuated.html

```diff
@@ -6088,6 +6092,21 @@ <h1>Acrobots, Cart-Poles, and Quadrotors</h1>
 
   </section>
 
+  <section data-type="sect1"><h1>Consider Jupyter Notebook</h1>
```

Comment:
cool -- agreed jupyter deserves a shoutout

as well exampled by the notebook Greg put up (http://people.csail.mit.edu/gizatt/DoublePendulumPassiveDynamics.html) the workflow is great

google colaboratory is good example of continuing momentum of jupyter notebooks

Reply:
the link is to the runnable version of greg's example.  (i just renamed it for consistency)